resources/cloudcontrol.go

```diff
@@ -0,0 +1,160 @@
+package resources
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/cloudcontrolapi"
+	"github.com/google/uuid"
+	"github.com/pkg/errors"
+	"github.com/rebuy-de/aws-nuke/pkg/types"
+	"github.com/sirupsen/logrus"
+)
+
+func init() {
+	// It is required to manually define Cloud Control API targets, because
+	// existing configs that already filter old-style resources could break,
+	// because the resource is also available via Cloud Control.
+	//
+	// To get an overview of available cloud control resource types run this
+	// command in the repo root:
+	//     go run ./dev/list-cloudcontrol
+	registerCloudControl("AWS::AppFlow::ConnectorProfile")
+	registerCloudControl("AWS::AppFlow::Flow")
+	registerCloudControl("AWS::AppRunner::Service")
+	registerCloudControl("AWS::ApplicationInsights::Application")
+	registerCloudControl("AWS::Athena::DataCatalog")
+	registerCloudControl("AWS::Backup::Framework")
+	registerCloudControl("AWS::MWAA::Environment")
+	registerCloudControl("AWS::Synthetics::Canary")
+	registerCloudControl("AWS::Timestream::Database")
+	registerCloudControl("AWS::Timestream::ScheduledQuery")
+	registerCloudControl("AWS::Timestream::Table")
+	registerCloudControl("AWS::Transfer::Workflow")
```

Comment:
I don't think it makes sense to keep a list, an advantage of CloudControl would be to immediately use a new resource after release because the APIs are the same. If we need to patch aws-nuke first it's not an advantage anymore. But I understand that we need it at the moment.

Reply:
So you do not agree with the reasoning I wrote in the comment above the list?

Reply:
I said I do understand it, but we should try to get rid of it in the future.

Reply:
IMO that will be only possible via a major release.